src/lender/tranche.sol

```diff
@@ -111,7 +111,7 @@ contract Tranche is Math, Auth, FixedPoint {
 
         users[usr].supplyCurrencyAmount = newSupplyAmount;
 
-        totalSupply = safeAdd(safeSub(totalSupply, currentSupplyAmount), newSupplyAmount);
+        totalSupply = safeAdd(safeTotalSub(totalSupply, currentSupplyAmount), newSupplyAmount);
```

Comment:
This is exactly where the problem occured on kovan :D

Reply:
I will take a deeper look